source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -21,9 +15,41 @@ Download **version 5.0 or greater** to ensure support for
 the :abbr:`AVX (Advanced Vector Extensions)` needed to run
 |CLOSIA|
 
-
 .. _create_vm_vbox:
 
+
+Add Windows, Linux distro, and MacOS here.
+
+Install VirtualBox
+===================
+#. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
+   `Intel®Virtualization Technology for Directed I/O`_ (Intel® VT-d) on the
+   host machine’s BIOS/UEFI.
```

Comment:
Recommend "Enable the Intel® Virtualization Technology (Intel® VT) and the Intel®Virtualization Technology for Directed I/O (Intel® VT-d) in the host machine’s BIOS."

Reply:
Done.